### PR TITLE
[NU-1740] preparing quickstart-setup to be extracted as an external image

### PR DESCRIPTION
**Pull request by mk-software-pl**
- mocks are part of the quickstart-setup container
- nu config is reloadable

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-NUSSKNACKER_VERSION=1.16.0
\ No newline at end of file
+NUSSKNACKER_VERSION=preview_NU-1740_scenario-types-reload_testing_1-latest
```
Comment (mk-software-pl) [inline .env:1]: I'm waiting for the merge of [this](https://github.com/TouK/nussknacker/pull/6591) to set here a snapshot version.
Currently, to test it, you have to build Nu locally:
```bash
bash -c "export NUSSKNACKER_SCALA_VERSION=2.12 && sbt clean dist/Docker/publishLocal"
```
before running this quickstart.

**Review by lukasz-bigorajski — APPROVED** [review]
I checked that it's working on my computer.
LGTM!

File: quickstart-setup/Readme.md [hunk 2]
```diff
@@ -3,4 +3,6 @@
 ## Resources:
 https://github.com/wiremock/wiremock-faker-extension/blob/main/docs/reference.md
 https://docs.wiremock.io/response-templating/basics/
-https://docs.wiremock.io/response-templating/dates-and-times/
\ No newline at end of file
+https://docs.wiremock.io/response-templating/dates-and-times/
+
+# todo: delete
```
[the remark this answers is not in the document]
Comment (mk-software-pl) [reply quickstart-setup/Readme.md:8]: it's a reminder for me. I'm going to write a readme in the next PR (in which I will introduce the docker image with quickstart-setup).

File: .env (hunk 1 above)
Comment (mk-software-pl) [reply .env]: ok, now snapshot staging-latest is configured

**Pull request merged**